R/vi_model.R

```diff
@@ -49,9 +49,9 @@ vi_model.C5.0 <- function(object, ...) {
   )
 
   # Add variable importance type attribute
-  dots <- list(...)
-  attr(tib, "type") <- if ("metric" %in% names(dots)) {
-    dots[["metric"]]
+  dot_pairlist <- substitute(...())
```

Comment:
We can get away list not evaluating all elements of `...` since we only need `metric` if it's there.